src/routes/appRouter.tsx

```diff
@@ -0,0 +1,119 @@
+import { createBrowserRouter } from 'react-router-dom';
+
+import MainPage from '@/pages/MainPage';
+import ProjectPage from '@/pages/project/ProjectPage';
+import ProjectDetailPage from '@/pages/project/ProjectDetailPage';
+import ProjectPostingPage from '@/pages/project/ProjectPostingPage';
+import ProjectUpdatePage from '@/pages/project/ProjectUpdatePage';
+import DeveloperListPage from '@/pages/developer/DeveloperListPage';
+import DeveloperDetailPage from '@/pages/developer/DeveloperDetailPage';
+import DMListPage from '@/pages/dm/DMListPage';
+import DMPage from '@/pages/dm/DMPage';
+import ProfilePage from '@/pages/profile/ProfilePage';
+import SettingsPage from '@/pages/settings/SettingsPage';
+import LoginPage from '@/pages/auth/LoginPage';
+import SignupPage from '@/pages/auth/SignupPage';
+import ExtraInformationPage from '@/pages/auth/ExtraInformationPage';
+import NotFoundPage from '@/pages/error/NotFoundPage';
+import NotificationPage from '@/pages/notification/NotificationPage';
+import HomePage from '@/pages/HomePage';
+
+export const router = createBrowserRouter([
+  {
+    path: '/',
+    element: <MainPage />,
+    errorElement: <NotFoundPage />,
+    children: [
+      {
+        index: true, // /페이지
+        element: <HomePage />,
+      },
+      {
+        path: 'projects',
+        children: [
+          {
+            path: '',
```

Comment:
좀 더 찾아보니까, index 를 사용하면 굳이 path 를 추가할 필요가 없네요~! ([참고](https://reactrouter.com/en/main/route/route#index))
```suggestion
            index: true,
```